src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Checklista</title>
+  <title>{{Checklista}}</title>
```

Comment:
Nie wiem czy to tylko żeby ładniej wyglądało ale jeśli tak to spoko. Jeśli nie to mam nadzieje, że wiesz o tym, że poza komponentami Angulara nie masz dostępu do takiego bindowania? ;)

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/18f30c8ce76f68e2f00c6864e255daee1c9f6576